payloads/external/tianocore/Kconfig

```diff
@@ -186,4 +186,28 @@ config TIANOCORE_USE_8254_TIMER
 
 endif
 
+config CONFIG_TIANOCORE_ENABLE_IPXE
```

Comment:
We laso need TIANOCORE_BOOTSPLASH_FILE and TIANOCORE_HAVE_EFI_SHELL to be outside of the `if TIANOCORE_UEFIPAYLOAD` above

Reply:
@miczyg1 I did that fix, but I have no idea what is the logic behind that. Even if these options are payload independent, then those should be under `UEFIPAYLOAD || COREBOOTPAYLOD` since enabling those without any payload doesn't make sense.